src/main/java/net/sf/jabref/logic/integrity/IntegrityCheck.java

```diff
@@ -18,18 +21,20 @@
     public static final Checker BRACKET_CHECKER = new BracketChecker();
```

Comment:
They are currently needed, as the caller cannot access the class directly because it is private.